FE/src/scss/_utils.scss

```
@@ -0,0 +1,137 @@
+@mixin button {
+  width: 13.4rem;
+  height: 4rem;
+  border-radius: .6rem;
+  border: 0;
+  font-size: $font-medium;
+  font-weight: $wgt-large;
+}
+
+@mixin button-blue($active) {
+  @include button;
+
+  @if $active {
+    background-color: $blue;
+    color: $white;
+  } @else {
+    background-color: $btn-accent-disabled;
+    color: $lightgray;
+  }
+}
+
+@mixin button-gray {
+  @include button;
+  background-color: $btn-normal-default;
+  color: $btn-normal-hover;
+}
+
+@mixin task($color: false, $box-sizing: false, $opactiy: 1, $background-color: $task-default) {
+  display: flex;
+  padding: 1.6rem;
+  opacity: $opactiy;
+  background-color: $background-color;
+  border-radius: .6rem;
+  box-shadow: 0 .1rem 3rem $task-box-shadow;
+  justify-content: space-between;
+  margin-bottom: 1.6rem;
+
+  @if $box-sizing {
+    box-sizing: border-box;
+  } @else {
+    box-sizing: content-box;
+  }
+
+  @if $color {
+    border: .1rem solid $color;
+  } @else {
+    border: 0;
+  }
+}
+
+@mixin task-input-exclude($btn-remove-img: true) {
+  &__contents {
+    cursor: pointer;
+    width: 100%;
+  }
+
+  &__title {
+    color: $task-title-color;
+    font-size: $task-title-fs;
+    font-weight: $task-title-wgt;
+    margin-bottom: .8rem;
+  }
+
+  &__desc {
+    color: $task-body-color;
+    font-size: $task-body-fs;
+    font-weight: $task-body-wgt;
+    margin-bottom: .8rem;
+  }
+
+  &__author {
+    color: $task-author-color;
+    font-size: $task-author-fs;
+    font-weight: $task-author-wgt;
+  }
+
+  @if $btn-remove-img {
+    &__btn--remove {
+      background-image: url('../assets/task-remove-default.svg');
+      background-repeat: no-repeat;
+      height: 1.2rem;
+      width: 1.2rem;
+    }
+  } @else {
+    &__btn--remove {
+      background-image: url('../assets/task-remove-delete.svg');
+      background-repeat: no-repeat;
+      height: 1.2rem;
+      width: 1.2rem;
+    }
+  }
+}
+
+@mixin task-input-include($background-color: true) {
+  flex-direction: column;
+
+  &__title {
+    margin-bottom: .8rem;
+  }
+
+  &__desc {
+    margin-bottom: 1.6rem;
+  }
+
+  input[type=text] {
```

nested 문법을 활용하면 더 깔끔하게 작성할 수 있을 것 같네요 
```suggestion
  input[type=text] {
    &:focus 
    }
```